src/h5web/providers/silx/api.ts

```diff
@@ -8,7 +8,17 @@ import {
   HDF5Collection,
   HDF5Values,
 } from '../models';
-import type { SilxMetadataResponse } from './models';
+import type { SilxEntity, SilxMetadataResponse } from './models';
+
+function getName(entity: SilxEntity): string {
+  const { alias } = entity;
+  if (alias.length === 0) {
+    return '';
+  }
+  const pathMembers = alias[0].split('/');
+
+  return pathMembers.length > 0 ? pathMembers[pathMembers.length - 1] : '';
+}
```

Comment:
In SilxAPI, I don't have an easy access to the link titles. 

I make use of the `alias` that is an array of the paths leading to the entity. The last part of the path is therefore the entity name.